internal/actioninfo/actioninfo.go

```diff
@@ -1,9 +1,29 @@
 package actioninfo
 
+import (
+	"fmt"
+	"log"
+)
+
 type DataParser interface {
 	// TODO: добавить методы
+	Parse(string) error
+	ActionInfo() (string, error)
 }
 
 func Info(dataset []string, dp DataParser) {
 	// TODO: реализовать функцию
+	for _, v := range dataset {
+		err := dp.Parse(v)
+		if err != nil {
+			log.Print("function Info: parsing method fault")
+			break
```

Comment:
Нужно перейти на следующую итерацию, а не останавливать совсем.
```suggestion
			continue
```